pkg/extensions/search/resolver.go

```diff
@@ -563,16 +574,20 @@ func (r *queryResolver) getImageList(store storage.ImageStore, imageName string)
 }
 
 func BuildImageInfo(repo string, tag string, manifestDigest godigest.Digest,
-	manifest v1.Manifest, imageConfig ispec.Image,
+	manifest v1.Manifest, imageConfig ispec.Image, isSigned bool,
 ) *gql_generated.ImageSummary {
 	layers := []*gql_generated.LayerSummary{}
 	size := int64(0)
-
 	log := log.NewLogger("debug", "")
-
 	allHistory := []*gql_generated.LayerHistory{}
-
 	formattedManifestDigest := manifestDigest.Hex()
+	annotations := common.GetAnnotations(manifest.Annotations, imageConfig.Config.Labels)
+
+	lastUpdated := imageConfig.Created
+
+	if lastUpdated == nil && len(imageConfig.History) > 0 {
```

Comment:
lastUpdated `0` value is not nil. Please see https://github.com/project-zot/zot/pull/822/files#diff-8e8bbae23fd292cf23636953113f80199da71bfd6c26e5a3074565cb7e0c0e3cR345